.rubocop.yml

```diff
@@ -9,3 +9,6 @@ AllCops:
 
 Style/Documentation:
   Enabled: false
+
+Metrics/BlockLength:
+  Enabled: false
```

Comment:
This is generally a good cop to keep enabled - it forces you to keep your methods small :) 

I tried in on local env and it seems like it throws only on rspec tests. Feel free to disable it only for `spec/**/**` path :)

Reply:
This was one of the topics I wanted to ask you about and now I know, thanks